Sources/Commands/SwiftPackageTool.swift

```diff
@@ -196,7 +196,10 @@ public class SwiftPackageTool: SwiftTool<PackageToolOptions> {
             // Toggle enable auto pinning if requested.
             if let enableAutoPin = options.pinOptions.enableAutoPin {
                 let workspace = try getActiveWorkspace()
-                return try workspace.pinsStore.load().setAutoPin(on: enableAutoPin)
+                let pinsStore = try workspace.pinsStore.load()
+                pinsStore.autoPin = enableAutoPin
+                try pinsStore.saveState()
```

Comment:
nit: since no value needs to be returned here, this can `return try pinsStore.saveState()`.